src/components/Book.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+
+const Book = function (props) {
+  const { book } = props;
+  return (
+    <li key={book.id}>
+      <span>{` ${book.title} `}</span>
+      <span>{` ${book.author} `}</span>
+      <button type="button">Remove book</button>
+    </li>
+  );
+};
+
+export default Book;
+
+Book.propTypes = {
+  book: PropTypes.shape({
+    id: PropTypes.number,
+    title: PropTypes.string,
+    author: PropTypes.string,
+  }).isRequired,
+};
```

Comment:
- [ ] Let us fix the following warnings

![testing-2](https://user-images.githubusercontent.com/55635977/145541907-94cb28a0-054a-4aee-becb-71460812aa8b.png)